spec/features/user_creates_a_dinosaur_spec.rb

```diff
@@ -0,0 +1,107 @@
+require 'rails_helper'
+
+feature 'user can create a new dinosaur', %{
+  As a user
+  I want to create a new dinosaur
+  So that I can share my favorite dinosaurs
+
+  Acceptance Criteria
+  [√] User can navigate to new dinosaur form from index
+  [√] User can view form properly
+  [√] User gets confirmation if dinosaur is entered correctly
+  [√] User gets errors if form is filled out incorrectly
+  [√] User gets redirected to show page of created dinosaur
+
+} do
+
+  feature "User is not signed in" do
+
```

Comment:
This is a bit awkward for me. The feature is that the user can create a new dinosaur, whereas the respective scenarios would be if they are signed in and if they are not.